Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10.16.0-alpine
+
+WORKDIR /workspace
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . ./
```

Comment:
Copy to `/workspace`? How do you think?

Reply:
See above for motivation. In general I was following the convention that was documented in https://nodejs.org/de/docs/guides/nodejs-docker-webapp/